assets/js/klarna-checkout-for-woocommerce.js

```diff
@@ -231,7 +231,17 @@ jQuery( function ( $ ) {
 						if ( name === "shipping_email" ) {
 							kco_wc.shippingEmailExists = true
 						}
-						$( "p#" + name + "_field" ).appendTo( "#kco-extra-checkout-fields" )
+
+						$( ".woocommerce-account-fields" ).appendTo( "#kco-extra-checkout-fields" )
```

Comment:
This is done inside a for loop of all fields. Wont this append the field for each item in the loop?

Reply:
If we know we want to move this entire wrapper each time, we could do that outside of the for loop, either before or after.